src/window.rs

```diff
@@ -132,6 +130,26 @@ impl Window {
         window
     }
 
+    /// Create the control panel
+    pub fn add_control_panel(&mut self) -> &WebView {
```

Comment:
Couldn't this just set in `new` method?

Reply:
Do you mean adding a param in `new` to control if we should create the control panel or not?

Reply:
yes